sw.js

```diff
@@ -4,7 +4,7 @@ https://gd-codes.github.io/mc-pixelart-maker/
 Cached site should require only ~ 3 MB space
 */
 
-const CURRENT_CACHE_VERSION = 'mapart-cache-4.5.0';
+const CURRENT_CACHE_VERSION = 'mapart-cache-4.6.0';
```

Comment:
Since this is a significant new change, increment the Service Worker version so that the changes can also reload to all users who have downloaded the PWA as an offline web-app.